pysidt/sidt.py

```diff
@@ -114,14 +116,14 @@ def generate_extensions(self, node, recursing=False):
         )
 
         if not out and not recursing:
-            logging.error("recursing")
-            logging.error(node.group.to_adjacency_list())
+            logging.info("recursing")
+            logging.info(node.group.to_adjacency_list())
             # for item in node.items:
-            #     logging.error(item.to_adjacency_list())
+            #     logging.info(item.to_adjacency_list())
```

Comment:
We should be able to just delete these commented out ones. I'm not even certain which of the ones that aren't commented out we should keep.